### DISCUSSION TITLE
All requested reviews are shown as assigned to you on notifications page

### DISCUSSION THREAD

**Discussion by moishinetzer** [community/community#47230] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone X (iOS 16.0)
</details>

![image](https://user-images.githubusercontent.com/11411486/218440824-f666976c-45d1-4684-a77e-4c09dd4af887.jpeg)

I was not requested to review and it does not show as a review when I click on the notification itself. However it shows assigned to me on the notifications page.

**Comment by mattruggio** [comment]
Hey @moishinetzer, I just want to confirm that the PR you are referencing did not happen to be assigned to your user account at one point in time, then was un-assigned.  I have noticed that if you are assigned, then un-assigned, the notification might still say "... requested your review on .." within it.  Can you verify the PR did not have any history, such as something like this in it?

**Reply by moishinetzer** [reply]
Hi @mattruggio I can confirm this is not the case. For all of the PR's that have anyone requested even without me being requested then removed the issue persists. 

Very interesting "other" account you have there 🤪

**Reply by mattruggio** [reply]
Thanks for confirming this @moishinetzer.  I found an existing bug on this in our backlog and I will be trying to bump it up in priority.

**Reply by moishinetzer** [reply]
Updates on this?

**Reply by mattruggio** [reply]
Unfortunately this specific bug has a few blockers we are working through before the fix can get rolled out. I can verify it is a higher priority reported bug but, as of today, there is no estimated completion date (yet).
